src/app/editor/viewer/lib/parsers/sketch-style-parser.service.ts

```diff
@@ -18,74 +19,91 @@ export enum BorderType {
 })
 export class SketchStyleParserService {
   constructor(private binaryPlistParser: BinaryPropertyListParserService) {}
-  public visit(pages: Array<SketchMSPage>) {
-    pages.forEach(page => {
+
+  visit(sketch: SketchData) {
+    sketch.pages.forEach((page: SketchMSPage) => {
       if (page.layers) {
-        page.layers.map(layer => this.visitObject(layer, page, layer));
+        page.layers.map((layer: SketchMSLayer) => {
+          this.visitObject(layer, page, layer);
+        });
       }
     });
   }
 
-  visitObject(obj: any, parent: any, root: any) {
-    for (const property in obj) {
-      if (obj.hasOwnProperty(property)) {
-        if (typeof obj[property] === 'object') {
-          // visit child
-          if (obj[property].frame && obj[property].frame._class === 'rect') {
-            this.visitObject(obj[property], obj, obj[property]);
+  /**
+   * Recurse over all layers properties and parse
+   * valuable class to CSS
+   */
+  visitObject(curr: any, parent: any, root: SketchMSLayer) {
+    for (const property in curr) {
```

Comment:
what's curr ?

Reply:
Oh, it was supposed to be obj, I'll change it.

Reply:
then just name it `currentObject` :)